source/file_management/file_management.rst

```diff
@@ -223,11 +223,52 @@ and ``build_systems``, for project-specific build systems.
 
 
 **Folders**
-    Each folder must have a ``path``,
-    and may optionally have a ``folder_exclude_patterns``
-and ``file_exclude_patterns`` setting.
-    The path may be relative to the project directory,
-    or an absolute path.
+    Each folder must have a ``path``, that may be absolute or
+    relative. In the latter case it will be relative to the
+    project directory.
+    
+    There are also the optional properties ``folder_include_patterns``,
+    ``file_include_patterns``, ``folder_exclude_patterns``
+    and ``file_exclude_patterns``. They all admit an array of strings
+    as values, and understand basic globbing syntax, such as "*.html".
+    
+    The ones that include files and folders, work by excluding
```

Comment:
You are referring to the "patterns" here, not the settings. While it can be interpreted to be the patterns, explicitly mentioning it is better. (It confused me at first.)